### ISSUE TITLE
Bundle chai & sinon

### ISSUE THREAD

**Issue by hazeledmands** [goodeggs/generator-goodeggs-npm#19]
...as these are almost always required while testing.

**Comment by hurrymaplelad** [comment]
We've got [mocha-sinon](https://github.com/goodeggs/generator-goodeggs-npm/commit/7e6e6cdd749a254bb305dd0bf26d776c3062d822) and [chai](https://github.com/goodeggs/generator-goodeggs-npm/commit/c8368de64bdc1d88b3e3b65d94708a096d5a9b2d) now, but we may still want [sinon-chai](https://github.com/domenic/sinon-chai).

**Comment by asalant** [comment]
Yep. Let's add that in. I'm down with making these our standard testing
toolkit.

On Wed, Feb 4, 2015 at 11:26 AM, Adam Hull <notifications@github.com> wrote:

> We've got mocha-sinon
> <https://github.com/goodeggs/generator-goodeggs-npm/commit/7e6e6cdd749a254bb305dd0bf26d776c3062d822>
> and chai
> <https://github.com/goodeggs/generator-goodeggs-npm/commit/c8368de64bdc1d88b3e3b65d94708a096d5a9b2d>
> now, but we may still want sinon-chai
> <https://github.com/domenic/sinon-chai>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/goodeggs/generator-goodeggs-npm/issues/19#issuecomment-72920874>
> .
>



-- 
eat well with good eggs on: twitter <http://twitter.com/goodeggs> ||
facebook <http://facebook.com/goodeggs> || blog <http://blog.goodeggs.com/>

**Comment by hurrymaplelad** [comment]
Cool.  Thoughts on including it directly vs via [goodeggs-spec-helpers](https://github.com/goodeggs/goodeggs-spec-helpers/blob/master/node_helper.coffee)? 

If we bundle goodeggs-spec-helpers by default, we also get `coffee` runnable specs and README documentation for best practices.  It'd be easiest to open-source spec-helpers.  We'd want to make sure requiring spec-helpers doesn't automatically pull in potentially unnecessary dependencies like karma.

**Comment by asalant** [comment]
I'd do it directly. I actually think the generator is worth open sourcing
in general - it's opinionated enough to be useful but not so much that
others won't use it (including our own teams). Adding goodeggs-spec-helpers
takes it too far in the not-relevant-to-others direction.

On Wed, Feb 4, 2015 at 12:08 PM, Adam Hull <notifications@github.com> wrote:

> Cool. Thoughts on including it directly vs via goodeggs-spec-helpers
> <https://github.com/goodeggs/goodeggs-spec-helpers/blob/master/node_helper.coffee>?
>
>
> If we bundle goodeggs-spec-helpers by default, we also get coffee
> runnable specs and README documentation for best practices. It'd be easiest
> to open-source spec-helpers. We'd want to make sure requiring spec-helpers
> doesn't automatically pull in potentially unnecessary dependencies like
> karma.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/goodeggs/generator-goodeggs-npm/issues/19#issuecomment-72928817>
> .
>



-- 
eat well with good eggs on: twitter <http://twitter.com/goodeggs> ||
facebook <http://facebook.com/goodeggs> || blog <http://blog.goodeggs.com/>
